src/cmd/terraform.go

```diff
@@ -27,9 +28,9 @@ func init() {
 }
 
 func newFile(filename string, makeExecutable bool) *os.File {
-	path := fmt.Sprintf("%s", filename)
+	path := filename
```

Comment:
this seems silly now - not sure why it was like this but we can probably refactor away the `path` variable and just use `filename` everywhere in this function.